cmd/subnetcmd/upgradecmd/vm.go

```diff
@@ -109,6 +109,20 @@ func upgradeVM(_ *cobra.Command, args []string) error {
 		return err
 	}
 
+	// if upgrading local, check that the network is off otherwise fail here
```

Comment:
probable also take network as off if server is off (depends on specific err return of binutils.NewGRPCClient())